flocker/testtools/_base.py

```diff
@@ -104,6 +111,153 @@ def mktemp(self):
         return make_temporary_directory(self).child('temp').path
 
 
+class _SplitEliotLogs(Fixture):
+    """
+    Split the Eliot logs out of Twisted logs.
+
+    Assumes that Twisted logs contain Eliot logs as per
+    ``flocker._redirect_eliot_logs_for_trial``, and that these logs have been
+    attached to a test case as a detail named with the value of
```

Comment:
It doesn't assume that they have been so attached, it ensure that they are itself.